code/modules/clothing/head/misc.dm

```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/head/canada
 	name = "striped red tophat"
-	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'érable</i>"
+	desc = "it smells like fresh donut holes / <i>il sent comme des trous de beignets frais</i>"

```

Comment:
"donut hole" is the not-trademarked way to say "Timbit" aka [these](https://41.media.tumblr.com/fee7c975646d599e5f3a915ade70a77e/tumblr_inline_o47nywagGM1rr9dpz_540.jpg). They are a bite-sized fried dough confectionery popular in my country. Also the sentence translates to french without having to use an accent character.